.github/workflows/ci.yml

```
@@ -47,6 +47,29 @@ jobs:
           echo "WASMEDGE_INCLUDE_DIR=C:\Program Files\WasmEdge\include" >> $env:GITHUB_ENV
       - name: Run checks
         run: make check
+
+  build-containerd-shim-wasm:
```

```suggestion
  build-all-features-containerd-shim-wasm:
```

or something like that to distinguish it from others?